lib/parser.js

```diff
@@ -789,6 +818,30 @@ function updateParseArgsForDefaultCapabilities (parser) {
   };
 }
 
+function parseSecurityFeatures (features) {
+  const splitter = (splitOn) => {
+    return (str) => {
+      return str.toString().split(splitOn).map(s => s.trim()).filter(Boolean);
+    };
+  };
+  let parsedFeatures;
+  try {
+    parsedFeatures = splitter(',')(features);
+  } catch (err) {
+    throw new Error('Could not parse value of --allow-insecure. Should be a list of strings separated by commas');
+  }
+
+  if (parsedFeatures.length === 1) {
+    // we might have a file which is a list of features
+    try {
+      const fileFeatures = fs.readFileSync(parsedFeatures[0], 'utf8');
+      parsedFeatures = splitter(os.EOL)(fileFeatures);
```

Comment:
I'd rather split on '\n' and then trim. It might be possible that on Windows we still have a file in Unix format

Reply:
👍